IntegrationTests/Helpers/String+NonceValidation.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+extension String {
+    var isANonce: Bool {
```

Comment:
I think generally our style has been to have a space between extension and the first declaration - this is something that we can also handle with a linter in the future.